src/main/java/uk/ac/bath/se/DungeonController.java

```diff
@@ -187,30 +205,46 @@ private void addRoomBounds(Dungeon dungeon, int roomQuantity) {
         final int LOWER_DISTANCE = 2;
         final int UPPER_DISTANCE = 6;
 
-        for (int room = 1; room <= roomQuantity; room++) { //1 - top left, 2 - top right, 3 - bottom left, 4 - bottom right
+        for (int room = 1; room <= roomQuantity; room++) { //1, - top left, 2 - top right, 3 - bottom left, 4 - bottom right
 
             switch (room) {
+                case 9:
+                case 5:
                 case 1:
-                    startHeight = rand.nextInt((dungeon.getHeight() / 2 - UPPER_DISTANCE) - LOWER_DISTANCE) + LOWER_DISTANCE;
-                    startWidth = rand.nextInt((dungeon.getWidth() / 2 - UPPER_DISTANCE) - LOWER_DISTANCE) + LOWER_DISTANCE;
+                    do {
+                        startHeight = rand.nextInt((dungeon.getHeight() / 2 - UPPER_DISTANCE) - LOWER_DISTANCE) + LOWER_DISTANCE;
+                        startWidth = rand.nextInt((dungeon.getWidth() / 2 - UPPER_DISTANCE) - LOWER_DISTANCE) + LOWER_DISTANCE;
+                    } while (roomCollision(startHeight,startWidth));
                     addGold(room, startHeight, startWidth);
                     break;
 
+                case 10:
+                case 6:
                 case 2:
-                    startHeight = rand.nextInt((dungeon.getHeight() / 2 - UPPER_DISTANCE) - LOWER_DISTANCE) + LOWER_DISTANCE;
-                    startWidth = rand.nextInt((dungeon.getWidth() - UPPER_DISTANCE) - (dungeon.getWidth() / 2 + LOWER_DISTANCE)) + (dungeon.getWidth() / 2 + LOWER_DISTANCE);
+                    do {
+                        startHeight = rand.nextInt((dungeon.getHeight() / 2 - UPPER_DISTANCE) - LOWER_DISTANCE) + LOWER_DISTANCE;
+                        startWidth = rand.nextInt((dungeon.getWidth() - UPPER_DISTANCE) - (dungeon.getWidth() / 2 + LOWER_DISTANCE)) + (dungeon.getWidth() / 2 + LOWER_DISTANCE);
+                    } while (roomCollision(startHeight,startWidth));
                     addGold(room, startHeight, startWidth);
                     break;
 
+                case 11:
```

Comment:
Why empty cases out of order, can a default case capture this instead

Reply:
the cases roll down to run the case below. This makes the rooms be put in the correct quadrant when there are more than four

Reply:
Add a comment saying intentional fall through because its often a bug when cases dont break

Reply:
https://google.github.io/styleguide/javaguide.html#s4.8.4-switch